### DISCUSSION TITLE
Is version change effect search text function in PDF

### DISCUSSION THREAD

**Discussion by crfreyr** [pymupdf/PyMuPDF#1660] [Q&A]
Hi ,

I have situation where I'm not able to search certain text in pdf with the latest version but with older version i.e. Version: 1.18.12 its searching the text .Is there anything I'm missing.

Here is my code:

text_instances = page.search_for(search)
for inst in text_instances:
        inst_text = page.get_text("text", clip=inst, flags=1)
        page.add_text_annot(inst.tr, t1)
        page.draw_rect(inst, color=col, fill=col, overlay=False)

**Comment by JorjMcKie** [comment]
I am not aware of specific changes. As you explicitely specify the flags ("preserve ligatures" and nothing else) the only way of explaining this is gone.
So please send me an example page ...
